get_mirrors.py

```diff
@@ -153,7 +153,7 @@ def is_json(my_json):
     """Check if json generated is valid."""
     try:
         json_object = json.loads(my_json)
-    except ValueError, e:
+    except ValueError as e:
```

Comment:
Here (and elsewhere) it looks like you're ignoring the value of the exception -- you can just delete the `as e`.